src/util.ts

```diff
@@ -2,13 +2,13 @@ export function isAPromise(obj: any): boolean {
   return obj.then && typeof obj.then === 'function'
 }
 
-export function quoteAndCombine(...args) {
+export function quoteAndCombine(...args: (string | number | boolean | any[])[]) {
```

Comment:
This typing might be a little clearer if you used the named Array type to avoid the parens and repeated brackets

```ts
Array<string | number | boolean | any[]>
```

Reply:
Done